run.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
+
+DATABASE_URL="${DATABASE_URL?Missing required env var: DATABASE_URL}"
+
+mkdir -p generated
+
+echo "** Building the JSON->NDJSON converter **"
+cargo build --release
+
+echo "** Downloading the latest market deals state **"
+curl --fail -o ./generated/StateMarketDeals.json.zst https://marketdeals.s3.amazonaws.com/StateMarketDeals.json.zst
+
+echo "** Converting from .json.zst to .ndjson **"
+./target/release/fil-deal-ingester ./generated/StateMarketDeals.json.zst > generated/StateMarketDeals.ndjson
+
+echo "** Parsing retrievable deals **"
+node scripts/parse-retrievable-deals.js
+
+echo "** Building the SQL query to update the SPARK DB **"
+node scripts/build-spark-update-sql.js
+
+echo "** UPDATING THE PRODUCTION DATABASE **"
+psql "$DATABASE_URL" -f generated/update-spark-db.sql
+
+echo "** DONE **"
+say "done"
```

Comment:
this will not work on linux but I'm not concerned about that now

Reply:
Yeah, I think that's fine - the script will do everything necessary, print `DONE`, and then fail on this last command. We can also remove it; I am no longer sure it's useful when the script took 3-5 hours to complete yesterday.

Reply:
I removed this line.